quantus/metrics/faithfulness_metrics.py

```diff
@@ -178,7 +180,14 @@ def __call__(
         # Asserts.
         assert_attributions(x_batch=x_batch_s, a_batch=a_batch)
 
-        for x, y, a in zip(x_batch_s, y_batch, a_batch):
+        # use tqdm progressbar if not disabled
+        if self.disable_progress_bar:
+            iterator = zip(x_batch_s, y_batch, a_batch)
+        else:
+            from tqdm import tqdm
```

Comment:
can you please put the imports at top of the scripts? the import should be placed before relative imports like import ..helpers.norm_func

Reply:
ah sorry, yes of course. (this was just a leftover from when I thought I would implement it as an optional package dependency. but as the use of tqdm is so widespread and the package is so tiny, I decided to just do this the disabled-by-default-way)